frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/vm-wrapper.ts

```diff
@@ -202,6 +211,26 @@ export class VMWrapper extends K8sResourceWrapper<VMKind, VMWrapper> implements
     return this;
   };
 
+  generateCloudInitPassword = () => {
```

Comment:
IMO, it is not a good idea to overload this wrapper with too over specific methods. 
We should not let this class to become huge and thus unmaintainable.

Can we rather?
1. keep the logic outside and call it only in default-template.ts
2. or move the logic into the VolumeWrapper where it makes more sense from responsibility standpoint